src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment, commentsType } from '../Comment';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.map(comment => (
+      <li
+        key={[comment.id]}
```

Comment:
```suggestion
        key={comment.id}
```
do you need brackets there?